packages/@aws-cdk/aws-ec2/lib/vpc.ts

```diff
@@ -1685,6 +1711,7 @@ class ImportedVpc extends VpcBase {
   public readonly isolatedSubnets: ISubnet[];
   public readonly availabilityZones: string[];
   public readonly vpnGatewayId?: string;
+  public readonly internetGateway: IInternetGateway = { internetGatewayId: "" };
```

Comment:
I'm unsure if I need to try to identify an existing `InternetGateway` from an imported/looked up VPC. Do I need to start looking into defining a `cxapi.IGW_PROVIDER`, or is there a better way?

Reply:
Make this a getter that throws, like this:

```
public get internetGateway(): IInternetGateway {
  throw new Error('Cannot access \'internetGateway\' of an imported VPC.');
}
```

Reply:
Ah, nice!